src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -107,7 +107,7 @@ class WorkspaceInvitePage extends React.Component {
      * @returns {Boolean}
      */
     getShouldShowAlertPrompt() {
-        return _.size(lodashGet(this.props.policy, 'errors', {})) > 0 || lodashGet(this.props.policy, 'alertMessage.length') > 0;
+        return _.size(lodashGet(this.props.policy, 'errors', {})) > 0 || lodashGet(this.props.policy, 'alertMessage').length > 0;
```

Comment:
I think this will actually cause the same problem as before 😅. If alertMessage isn't there it'll be calling length on undefined. I think we need just a default value to be the third parameter of the lodashGet function.

Reply:
Just ran this with a new account and it looks like alertMessage will be set as an empty string. I think I'll just default it to an empty string just in case?
<img width="1245" alt="Screen Shot 2021-10-08 at 5 15 33 PM" src="https://user-images.githubusercontent.com/49007721/136633831-57f4feaa-60fe-4058-844e-4c02d60c3188.png">